spreadsheets/spreadsheet-updater.ts

```diff
@@ -0,0 +1,27 @@
+import { SpreadSheet } from './spreadsheet';
```

Comment:
A bit nitpicky, but since spreadsheet is a single word, please change the casing everywhere
`SpreadSheet`  --> `Spreadsheet`
`spreadSheetUpdater` --> `spreadsheetUpdater`